config/locales/es.label.yml

```diff
@@ -115,3 +115,9 @@ es:
       subtitle: 'antes: Barcelona on Rails'
       meta_desc: Grupo de usuarios de Ruby en Barcelona
       custom_recurrence: Tercer jueves de cada mes a las 19:00
+    dresdenrb:
+      :city: City
```

Comment:
Please provide correct values for Spanish and Polish too :)

Reply:
Done, thank you.